tools/coqdoc/output.ml

```diff
@@ -655,10 +656,23 @@ module Html = struct
 
   let reference s r =
     match r with
-    | Def (fullid,ty) ->
+    | Def [fullid,ty] ->
         let s' = sanitize_name fullid in
         printf "<a id=\"%s\" class=\"idref\" href=\"#%s\">" s' s';
         printf "<span class=\"id\" title=\"%s\">%s</span></a>" (type_name ty) s
+    | Def l ->
+        let ty =
+          l |> List.map (fun (_,ty) -> type_name ty)
+          |> CList.sort_uniquize String.compare
+          |> String.concat ", " in
+        printf "<span class=\"id\" title=\"%s\">" ty;
```

Comment:
Looking at the HTML output, I notice that we are losing a useful feature. The user can no longer get a link to an inductive by clicking on the symbol, since there is no `<a href="...">` left. So, it would be great if one of the `<span>` (ideally, the one corresponding to the inductive type itself) was output as a `<a href="...">`.